### DISCUSSION TITLE
Allow raids in claims with /accesstrust, or make a new command like /claimraids

### DISCUSSION THREAD

**Discussion by c0wg0d** [GriefPrevention/GriefPrevention#1579] [Ideas]
I made a raid farm that I would like to give people access to, but as soon as they enter the claim for the raid to start, they lose the bad omen effect.  Making a subdivision and granting /accesstrust or /containertrust doesn't allow them to start a raid--only /trust does.  I don't want people breaking blocks or opening chests.

Please either allow raids in claims with only /accesstrust, or make a new command like /claimraids that allows raids to happen regardless of who is trusted in the claim (similar to how /claimexplosions works).

**Comment by Jikoo** [comment]
There is a config option to not require trust to start raids, `GriefPrevention.Claims.RaidTriggersRequireBuildTrust`. I know it doesn't exactly resolve your situation, namely only allowing a specific subset to start raids, but it does at least allow raids without trust.
